server/src/utils/enums/ErrorMessages.ts

```diff
@@ -16,14 +16,15 @@ export enum ErrorMessages {
 	PASSWORDS_DO_NOT_MATCH_ERROR_MESSAGE = 'Passwords do not match',
 
 	// Validation exists
-	USERNAME_ALREADY_REGISTERED_ERROR_MESSAGE = 'Username is already in use',
-	EMAIL_ALREADY_REGISTERED_ERROR_MESSAGE = 'Email is already in use',
+	USERNAME_ALREADY_REGISTERED_ERROR_MESSAGE = 'This username is already taken',
+	EMAIL_ALREADY_REGISTERED_ERROR_MESSAGE = 'Another account already uses this email address',
 	MEMBER_ALREADY_EXISTS_ERROR_MESSAGE = 'Member already exists',
 
 	// Forgot Password
 	EMAIL_NOT_FOUND_ERROR_MESSAGE = 'This account does not exist. Please try again.',
 
 	// Validation Args
 	USERNAME_MUST_BE_LONG_ERROR_MESSAGE = 'The username must be at least 3 character long.',
-	PASSWORD_FORMAT_ERROR_MESSAGE = 'The password must have at least 8 characters, one upper case, one lower case, one number and one special character.'
+	PASSWORD_FORMAT_ERROR_MESSAGE = 'The password must have at least 8 characters, one upper case, one lower case, one number and one special character.',
+	CONFIRMED_PASSWORD_ERROR_MESSAGE = 'Passwords did not match.'
```

Comment:
I don't know if we need a new ErrorMessages when we have `PASSWORDS_DO_NOT_MATCH_ERROR_MESSAGE` in the same file.